src/annotator/test/sidebar-test.coffee

```diff
@@ -19,15 +19,31 @@ describe 'Sidebar', ->
     return new Sidebar(element, config)
 
   beforeEach ->
+    sandbox.stub(Sidebar.prototype, '_setupGestures')
```

Comment:
This was a workaround for some issue I think. Can you remember what that was? A brief comment to that effect would be useful.